CMakeLists.txt

```diff
@@ -169,32 +169,46 @@ foreach(HEADER_SRCPATH IN ITEMS ${AWS_COMMON_HEADERS} ${AWS_COMMON_OS_HEADERS} $
 		message(ERROR "Couldn't find source root for header ${HEADER_SRCPATH}")
 	endif()
 
-    install(FILES ${HEADER_SRCPATH} DESTINATION ${HEADER_DSTDIR})
+    install(FILES ${HEADER_SRCPATH}
+            DESTINATION ${HEADER_DSTDIR}
+            COMPONENT Development)
 endforeach()
 
-set(LIBRARY_DIRECTORY ${CMAKE_INSTALL_PREFIX}/lib)
```

Comment:
The `CMAKE_INSTALL_PREFIX` is implicit when specifying `DESTINATION` in `install()`, so it's normally better to leave it off for packaging, as specifying it will cause stuff to get installed to `/usr/local/` instead of `/usr/`.

Reply:
Good point.